### ISSUE TITLE
Phone stops being detected after some usage, only charges afterwards.

### ISSUE THREAD

**Issue by papaustin** [Genymobile/gnirehtet#185]
By this I mean my PC does not recognize my phone at all, on the other side, my phone does not show the select USB Configuration prompt, it only charges. The only fix is to reboot my PC and this is temporary.

**Comment by rom1v** [comment]
On Windows?

Also, you should try with another USB cable.

**Comment by papaustin** [comment]
Yes, 64-bit Windows.
Ive tried multiple cables and the issue happens with all of them.
The issue happens during heavy load and also during no load at all.

**Comment by rom1v** [comment]
Probably a problem on the device itself, then. :disappointed:

**Comment by papaustin** [comment]
https://github.com/Genymobile/gnirehtet/issues/162#issuecomment-448235591
I read through some of the issues and it seems its not isolated to my device. This guy had the exact same issue as myself.

**Comment by rom1v** [comment]
OK. The comment you linked suggests it happened when they updated the ROM (so a problem on the device).

Anyway, it is a USB or adb problem. Unfortunately, probably nothing that can be solved in scrcpy.

**Comment by papaustin** [comment]
Yeah I guessed so. Weird thing as it used to work perfectly, and I hadnt have made any changes to the ROM in the past weeks.

**Comment by papaustin** [comment]
I'm updating this issue to comment that I did a full ROM wipe and reinstall.  Same issue as before.

**Comment by rom1v** [comment]
When it fails, does the device appear in `adb devices`?

**Comment by papaustin** [comment]
No, even killing adb and restarting it doesn’t work for the phone to be identified again.
On May 24, 2019, 12:17 AM -0500, Romain Vimont <notifications@github.com>, wrote:

When it fails, does the device appear in adb devices?

**Comment by papaustin** [comment]
No, even killing adb and restarting it doesn’t work for the phone to be identified again.
